packages/frontend-core/src/components/grid/cells/RelationshipCell.svelte

```diff
@@ -478,4 +531,19 @@
   .search :global(.spectrum-Form-item) {
     flex: 1 1 auto;
   }
+
+  .relationship-fields {
+    margin: var(--spacing-m) var(--spacing-l);
+    display: grid;
+    grid-template-columns: auto auto;
+    grid-row-gap: var(--spacing-m);
+    grid-column-gap: var(--cell-spacing);
+  }
+
+  .values:not(.wrap) .badge.extra-info {
+    cursor: pointer;
+  }
+  .values:not(.wrap) .badge.extra-info:hover::before {
+    opacity: 0.7;
```

Comment:
NAB: instead of using a pseudo ::before element and changing the opacity like this, it would be simpler (and more performant) to add something like `filter: brightness(1.2)` to the badge on hover.

Reply:
I did not know about this css API. Much easier :)